sovtokenfees/sovtokenfees/static_fee_req_handler.py

```diff
@@ -42,8 +42,15 @@ class StaticFeesReqHandler(FeeReqHandler):
     state_serializer = JsonSerializer()
 
     def __init__(self, ledger, state, token_ledger, token_state, utxo_cache,
```

Comment:
Please make sure that we call super() for `updateState`. Otherwise indy-node's logic of writing auth rules to config state will be broken.

Reply:
Also, we should not overwrite write_types and query_types attributes. Instead of we should append new txn_types with saving ConfigReqHandler's write_types/query_types sets.
For example, in this case you can use the next instruction:
write_types = FeeReqHandler.write_types.union({SET_FEES, FEE_TXN})
query_types = FeeReqHandler.query_types.union({GET_FEES, })

Also, valid_txn_types attribute is not needed, because already we have valid_txn_types property in parent LedgerRequestHandler class which is union of write_types and query_types sets